src/Controls/samples/Controls.Sample.UITests/Issues/Issue18857.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue18857">
+  <VerticalStackLayout>
+    <ImageButton 
+      x:Name="BackgroundImageButton"
+      AutomationId="TestImageButton"
+      HorizontalOptions="Center" 
+      Aspect="AspectFit" 
+      BorderColor="Red"
+      BorderWidth="10"
+      CornerRadius="12"
```

Comment:
Seems that the CornerRadius is not working 
![image](https://github.com/dotnet/maui/assets/6755973/a164f140-2b82-43b1-b09b-887b89d3c30b)